components/create_post/create_post.tsx

```diff
@@ -602,7 +594,7 @@ class CreatePost extends React.PureComponent<Props, State> {
         // Posting a message completes the tip when there are prewritten messages.
         // We do not complete messages in the control group,
         // so as to not alter behavior in the control group as a result of the A/B test code changes.
```

Comment:
Can we remove this comment on lines 602-604? I think that without the extra A/B test logic, the code is clear enough on its own.